src/audio/volume/volume.c

```diff
@@ -95,6 +88,13 @@ static enum task_state vol_work(void *data)
 		return SOF_TASK_STATE_COMPLETED;
 	}
 
+	/* The first is set and cleared to indicate ongoing ramp, the
+	 * latter is set once to enable self launched ramp only once
+	 * in stream start.
+	 */
+	cd->vol_ramp_active = true;
+	cd->ramp_started = true;
```

Comment:
This is mistake I made in rebase, I need to move this to other commit.

Reply:
This is fixed now. This chunk is now in commit "Volume: Always launch ramp in beginning of copy() operations".